roles/matrix-etherpad/defaults/main.yml

```diff
@@ -51,7 +51,12 @@ matrix_etherpad_database_connection_string: 'postgres://{{ matrix_etherpad_datab
 
 # Variables configuring the etherpad
 matrix_etherpad_title: 'Etherpad'
-matrix_etherpad_default_pad_text: 'Welcome to Etherpad!\n\nThis pad text is synchronized as you type, so that everyone viewing this page sees the same text. This allows you to collaborate seamlessly on documents!\n\nGet involved with Etherpad at https:\/\/etherpad.org\n'
+matrix_etherpad_default_pad_text: |
```

Comment:
Should we change the default text?

Reply:
This default text seems nice and descriptive to me.